chat-widget/src/components/startchaterrorpanestateful/StartChatErrorPaneStateful.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Dispatch, useEffect } from "react";
+import { IStyle, IImageProps } from "@fluentui/react";
+
+import { ILiveChatWidgetAction } from "../../contexts/common/ILiveChatWidgetAction";
+import { ILiveChatWidgetContext } from "../../contexts/common/ILiveChatWidgetContext";
+import { ILoadingPaneControlProps } from "@microsoft/omnichannel-chat-components/lib/types/components/loadingpane/interfaces/ILoadingPaneControlProps";
+import { ILoadingPaneStyleProps } from "@microsoft/omnichannel-chat-components/lib/types/components/loadingpane/interfaces/ILoadingPaneStyleProps";
+import { LoadingPane } from "@microsoft/omnichannel-chat-components";
+import { findAllFocusableElement } from "../../common/utils";
+import useChatContextStore from "../../hooks/useChatContextStore";
+import { TelemetryHelper } from "../../common/telemetry/TelemetryHelper";
+import { LogLevel, TelemetryEvent } from "../../common/telemetry/TelemetryConstants";
+import { defaultErrorPaneGeneralStyleProps } from "./common/defaultErrorPaneGeneralStyleProps";
+import { defaultErrorPaneTitleStyleProps } from "./common/defaultErrorPaneTitleStyleProps";
+import { defaultErrorPaneSubtitleStyleProps } from "./common/defaultErrorPaneSubtitleStyleProps";
+import { defaultErrorPaneIconStyleProps } from "./common/defaultErrorPaneIconStyleProps";
+import { defaultErrorPaneIconImageStyleProps } from "./common/defaultErrorPaneIconImageProps";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const StartChatErrorPaneStateful = (props: any) => {
+    const [state, ]: [ILiveChatWidgetContext, Dispatch<ILiveChatWidgetAction>] = useChatContextStore();
+    const { errorPaneProps, startChatErrorPaneProps } = props;
+    
+    const generalStyleProps: IStyle = Object.assign({}, defaultErrorPaneGeneralStyleProps, errorPaneProps?.styleProps?.generalStyleProps);
+    const titleStyleProps: IStyle = Object.assign({}, defaultErrorPaneTitleStyleProps, errorPaneProps?.styleProps?.titleStyleProps);
+    const subtitleStyleProps: IStyle = Object.assign({}, defaultErrorPaneSubtitleStyleProps, errorPaneProps?.styleProps?.subtitleStyleProps);
+    const iconStyleProps: IStyle = Object.assign({}, defaultErrorPaneIconStyleProps, errorPaneProps?.styleProps?.iconStyleProps);
+    const iconImageProps: IImageProps = Object.assign({}, defaultErrorPaneIconImageStyleProps, errorPaneProps?.styleProps?.iconImageProps);
+
+    const errorUIStyleProps: ILoadingPaneStyleProps = {
```

Comment:
Seems like we are purely using `errorPaneProps` for styles and `startChatErrorPaneProps` for controls (strings). Is that by design?

The reason behind this is: will it cause confusion - if I add props for `errorPaneProps.controlProps`, it actually doesn't do anything right?

Reply:
Did some refactoring -- `errorPaneProps` is now consolidated into `startChatErrorPaneProps`, with `startChatErrorPaneProps.controlProps` controlling all the associated props with this new pane. 

Additional `StartChatErrorPane` error string scenarios will be added within `startChatErrorPaneProps.controlProps` such as `authTitleText`, `authSubtitleText`, `[scenario]TitleText`, `[scenario]SubtitleText`.